### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Microsoft IIS Home Page_Link: "Not Connected" link is not descriptive and there is no programmatically determined meaningful link context.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#68]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA

**MAS ID Violated**: MAS2.4.4

**Repro Steps:**
1. Open "Microsoft IIS" Home Page using the URL: https://manage.iis.net/
2. Verify that all links in the page is unique and descriptive.
3. Tab till "Not Connected" link and verify that screen reader is reading its programmatically determined context to make the link descriptive for screen reader users.

**Actual Result:**
"Not Connected" link is not descriptive and the purpose of the link cannot be determined from the link text alone. 
There is no programmatically determined meaningful link context for screen reader users. Even tooltip is not sufficient to explain the context properly.

**Expected Result:**
Link text should identify the purpose of the link without needing additional context. 
There should be a programmatically determined meaningful context to make the link descriptive for screen reader users.
Link text should be like: "Connect to Server", tooltip should be like: "Not Connected", so that user can know that currently application is not connected and if user will select "connect to server" link, it will navigate them to the page from where they can connect the application to the server.

**User Impact:**
This Success Criterion helps people with motion impairment by letting them skip links that they are not interested in, avoiding the keystrokes needed to visit the referenced content and then returning to the current content.

MAS Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7}

**Attachments:**
![bug 68_not connected link is not descriptive](https://user-images.githubusercontent.com/38521133/44571996-24466800-a7a0-11e8-8fe9-d0cf7f8f064e.png)
[Bug 68_Not connected link is not descriptive.pptx](https://github.com/Microsoft/IIS.WebManager/files/2317502/Bug.68_Not.connected.link.is.not.descriptive.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by v-lasiva** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #C+E-H; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #MAS2.4.4; #Keyboard; #.NETWebsites; #Edge; #Chrome

**Comment by fsteffi** [comment]
#A11ySev2;

**Comment by fsteffi** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will  be removed.
